test.js

```diff
@@ -199,9 +234,38 @@ describe('capture-exit', function() {
       });
     });
 
+    it('throws an exit code of the first registered handler which rejects', function() {
```

Comment:
@stefanpenner If I understand you correctly, you mean a separate acceptance(`childProcess.execSync()`) test here. Right?